gui/wxpython/nviz/wxnviz.py

```diff
@@ -2443,3 +2448,16 @@ def GetCmd(self):
 
     def Corresponds(self, item):
         return sorted(self.GetCmd()) == sorted(item.GetCmd())
+
+
+__all__ = [
+    "DM_FLAT",
+    "DM_GOURAUD",
+    "DRAW_QUICK_SURFACE",
+    "DRAW_QUICK_VLINES",
+    "DRAW_QUICK_VOLUME",
+    "DRAW_QUICK_VPOINTS",
+    "MAX_ISOSURFS",
+    "ImageTexture",
+    "Nviz",
+]
```

Comment:
As a general guidance, what guided your choices to select what we want to expose to star imports when using from imports of that file?

Reply:
No "general guidance" here I am afraid, these  are objects imported in wxnviz that are used in other files, this was solving imported but unused.